content/blog/discover-the-power-of-data-center-monitoring-using-redfish-telemetry-and-cloud-native-tooling.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Discover the power of data center monitoring using Redfish telemetry and
+  cloud-native tooling
+date: 2023-10-05T17:03:36.464Z
+author: Naveen Gupta
+authorimage: /img/photo-jpg-ng.jpg
+thumbnailimage: ""
+disable: false
+tags:
+  - ilo-restful-api
+  - Grafana
+  - Telemetry
+  - monitoring
+  - Prometheus
+  - Redfish
+  - Alertmanager
+---
+<style>
+ul li{
+ font-size:27px;
+}
+</style>
+
+Monitoring data center infrastructure is critical for ensuring optimal performance, resource utilization, and timely issue detection. Redfish, an open industry standard for hardware management, provides a standardized way to access telemetry data from servers and other devices. Coupling Redfish telemetry with cloud-native monitoring tools offers a robust solution for real-time monitoring, data analysis, visualization, and alerting. 
+
+## **Why does data center monitoring matter?**
+
+Data centers are the backbone of modern businesses, housing critical applications, databases, and services. Ensuring the seamless operation of these data centers is essential for business continuity and meeting the demands of today's digital world. Data center monitoring helps with following:
+
+* **Performance Optimization:** Monitoring helps identify bottlenecks, inefficiencies, and potential failures, allowing for proactive optimization.
+* **Resource Utilization:** Tracking resource usage ensures that capacity is allocated effectively, saving costs and energy.
+* **Predictive Maintenance:** Real-time insights enable predictive maintenance, reducing downtime and associated costs.
+* **Compliance:** Monitoring helps meet regulatory requirements by maintaining accurate records and ensuring security.
+
+## **A comprehensive approach**
+
+My latest technical whitepaper, *“[Data center monitoring using Redfish telemetry and cloud-native tooling](https://www.hpe.com/psnow/doc/a00134351enw)”*, presents a comprehensive approach to data center monitoring by integrating the Redfish telemetry with cloud-native open-source tools including Telegraf, Prometheus, Alertmanager, and Grafana. These tools work seamlessly together to provide a holistic view of your data center infrastructure. The telemetry data source for this stack is HPE Integrated Lights-Out (iLO) which exposes metrics via Redfish interface.
+
+Here's a glimpse of what you'll discover in the whitepaper:
+
+* **Understanding Redfish Telemetry:** Learn about the Redfish standard and how it simplifies hardware telemetry data collection.
```

Comment:
Similar to the previous set of bullets, please use sentence case so that the terms "Telemetry", "-Native", and "Tools" all have initial lower case letters (telemetry, -native, tools).

In addition, if you can provide some space between the bullets for "breathing room", that would be great and make it easier for the reader to consume:

![Uploading image.png…]()

Reply:
Fixed